### ISSUE TITLE
Could not validate credentials - Everyone is locked out

### ISSUE THREAD

**Issue by hanserasmus** [mealie-recipes/mealie#2024]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [X] I have checked for existing issues that have been resolved in v1-beta

### What is the issue you are experiencing?

Everything was working fine yesterday morning. This morning, trying to log into the front-end with one of the 2 accounts on the instance, both says Could Not Validate Credentials. I also cannot use my API token anymore, since that too now says Could not validate. I changed NOTHING since yesterday. Why would this happen. There is nothing in the mealie.log besides some scraping errors about missing images (which is probably because of my pihole rules, so completely different issue).

Upgrading at the moment to 1.0 is not an option, I first need to make sure this particular instance of mine is working properly.

### Deployment

Docker (Linux)

### Mealie Version

0.5.6

**Comment by hanserasmus** [comment]
Upon further investigation, it seems that I am able to log in with the default user and password, but ALL of the setups that I have done, is gone. No users, no recipes, nothing. And I checked, the docker container was running for 35 hours and was healthy. So it is not something I did or changed.

**Comment by hay-kot** [comment]
Not sure how I can help you without deployment details or logs. Please provide.

- Container Logs
- Deployment Details (version, docker host os, etc...)
- docker-compose file.

**Comment by hanserasmus** [comment]
Hi @hay-kot . Thank you for the quick response.

Deployment is done without docker-compose file. I did:

```
mkdir /home/user/meailie
docker run -d --restart=unless-stopped -p 9925:80 -v `pwd`:'/app/data' hkotel/mealie:latest
```
It created all the folders and files by itself.

I managed to fix this now it seems. For some reason docker started by mounting the volumes wrong, I have no idea why that would be, but forcing it to use the original folder as the volume mount (instead of `pwd`) seemed to fix it. I was still not able to use my own account, but my wife's account was working, and I made her an admin luckily, so I was able to use her account to fix my password, and I am back in.

Sorry for the inconvenience caused.
